tests/apollo/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ foreach(STORAGE_TYPE ${STORAGE_TYPES})
           WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
   add_test(NAME skvbc_reconfiguration_${STORAGE_TYPE} COMMAND sh -c
-          "env ${APOLLO_TEST_ENV} STORAGE_TYPE=${STORAGE_TYPE} python3 -m unittest test_skvbc_reconfiguration ${TEST_OUTPUT}"
+          "env ${APOLLO_TEST_ENV} LEAKCHECK=ON STORAGE_TYPE=${STORAGE_TYPE} python3 -m unittest test_skvbc_reconfiguration ${TEST_OUTPUT}"
```

Comment:
I've added the LEAKCHECK=ON because, in order to clean the metadata storage, the replicas need to be shutdown normally.
Apollo is calling to p.kill unless LEACKCHECK=ON and then it calls to p.terminate

Reply:
@HristoStaykov what do you think about this?

Reply:
I added this because for LeakSanitizer reports we needed to terminate the Replica processes with SIGTERM instead of SIGKILL. This adds slow down for the tests, so at the time I was integrating LeakCheck to Apollo tests, I couldn't think of other uses for it. Now that we have them, I propose to make a separate option, might be GRACEFUL_SHUTDOWN for Apollo to consider when it has to terminate a Replica process, and I will add its usage when we have LEACKCHECK=ON. What do you guys think @teoparvanov @yontyon ?

Reply:
Yes, I will prepare the changes, for now `LEAKCHECK=ON` passed as environment variable will only have effect on termination of Replicas.

Reply:
https://github.com/vmware/concord-bft/pull/884